wheel/src/api.rs

```diff
@@ -14,9 +15,14 @@ use pyo3::prelude::*;
 use pyo3::types::{PyBytes, PyTuple};
 use pyo3::wrap_pyfunction;
 
+fn eval_to_py(err: EvalErr) -> PyErr {
```

Comment:
this looks somewhat similar to `adapt_response()`. do we need both?

Reply:
yeah one converts using the context to create a proper py response, the other just maps the existing rust error